caveclient/chunkedgraph.py

```diff
@@ -1361,6 +1398,7 @@ def get_past_ids(
         endpoint_mapping = self.default_url_mapping
 
         params = {}
+        # TODO should anything happen with client timestamps here?
```

Comment:
should get_past_ids have a default cutoff of the timestamp for the version?

Reply:
The current version only returns data if timestamps are explicitly set, so I think this one doesn't change either.